src/app/static/src/app/types.ts

```diff
@@ -27,4 +28,21 @@ export type IndicatorValuesDict= Dict<IndicatorValues>;
 export interface LocalSessionFile {
     hash: string
     filename: string
+}
+
+export interface BarchartIndicator {
+    indicator: string,
+    value_column: string,
+    indicator_column: string,
+    indicator_value: string,
+    name: string,
+    error_low_column: string,
+    error_high_column: string
+}
+
+export interface Filter {
+    id: string,
+    column_id: string,
```

Comment:
column_id is here because, as things stand with the filters returned in modelRunResults, their identifiers (e.g. 'age') don't actually correspond to the columns in which data is found, so we'll need both id (to find the filter options) and column_id (to find the data). This could be changed to avoid need for both, but would be a breaking change for the choropleth filters.